devito/functions/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .constants import *  # noqa
```

Comment:
I'd call it "constant", not "constants"

Reply:
Well for now, but may have constant tensors, ... at some points so won't be just this one.